src/dataflow/src/server.rs

```diff
@@ -100,6 +102,17 @@ pub enum SequencedCommand {
         id: GlobalId,
         /// A list of updates to be introduced to the input.
         updates: Vec<Update>,
+        /// If Some, a channel containing the results of durably storing the
+        /// transaction's writes to permanent storage. The receiver side of this
+        /// channel should be drained until sender hangup before returning
+        /// success to the user and an error returned if any of the received
+        /// responses are Errs.
+        ///
+        /// The receiver side of this is allowed to hang up (e.g. if the pgconn
+        /// that executed the INSERT is severed before the write finishes
+        /// persisting) and errors resulting from that are ignored by the
+        /// sender.
+        tx: Option<mpsc::UnboundedSender<Result<(), String>>>,
```

Comment:
Frank has banned this from Slack: "Now hear this: no new mpsc queues in the dataflow crate!"

Reply:
However I'm not sure what they're supposed to be replaced with -_- so :shrug:

Reply:
Yup! I'm waiting to see what the blessed alternative is